src/components/filters/component.js

```diff
@@ -15,40 +15,49 @@ import { fetchRedListCategories } from 'services/red-list';
 
 import './styles.scss';
 
-const Filters = ({ activeFilters, onClick }) => {
+const Filters = ({ filters, setFilters, resetFilters, onClick }) => {
 
   const [families, setFamilies] = useState([]);
   const [publications, setPublications] = useState([]);
   const [conservationFrameworks, setFrameworks] = useState([]);
   const [flyways, setFlyways] = useState([]);
   const [redList, setListCategories] = useState([]);
-
   useEffect(() => {
     fetchFamilies().then(data => setFamilies(data));
     fetchPublications().then(data => setPublications(data));
     fetchConservationFrameworks().then(data => setFrameworks(data));
     fetchFlyways().then(data => setFlyways(data));
     fetchRedListCategories().then(data => setListCategories(data));
-  }, [families, publications, conservationFrameworks, flyways, redList]);
+  }, []);
 
   const handleClick = () => {
-    onClick(false);
-  }
-
+    onClick();
+    resetFilters();
+  };
+
+  // filters values
+  const selectedFamily = filters.family;
```

Comment:
Are these names correct? I mean, you declared in the initialState filters.family_id, filters.redlist_id... but here I don't see the *_id after every name. Is this working?